### PR TITLE
Add Dockerfile highlighting support

### PR DESCRIPTION
**Pull request by kburton-dev**
Based on keywords listed here: https://docs.docker.com/reference/dockerfile/#overview

With support for `AS` keyword when aliasing image names, and highlighting image names and tags:

image

### CODE REVIEW

File: src/Languages/Dockerfile/DockerfileLanguage.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageAliasNamePattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageNamePattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\ImageTagPattern;
+use Tempest\Highlight\Languages\Dockerfile\Patterns\KeywordPattern;
+
+class DockerfileLanguage extends BaseLanguage
+{
+    public function getName(): string
+    {
+        return 'Dockerfile';
```
Comment (brendt) [inline src/Languages/Dockerfile/DockerfileLanguage.php:18]: Should this be `dockerfile`? Maybe add `docker` as an alias as well?

**Comment by brendt** [comment]
Let me know your thoughts about that one comment, otherwise I'm good merging it 👍

File: src/Languages/Dockerfile/DockerfileLanguage.php (hunk 1 above)
Comment (kburton-dev) [reply src/Languages/Dockerfile/DockerfileLanguage.php]: I had the same thought as you regarding the alias. And ChatGPT would also describe it this way, so I think `dockerfile` and `docker` will be enough.

**Comment by brendt** [comment]
Cool, thanks!

**Comment by brendt** [comment]
Tagged: https://github.com/tempestphp/highlight/releases/tag/2.8.0

**Pull request merged**